### PR TITLE
Clean up use of find/delete for CVE

### PR DESCRIPTION
**Pull request by pdmack**
Minor style improvement

### CODE REVIEW

File: models/mlflow/docker/Dockerfile [hunk 1]
```diff
@@ -33,7 +33,7 @@ RUN sed -i 's/conda activate base/conda activate mlflow/g' ~/.bashrc
 SHELL ["/opt/conda/bin/conda", "run", "-n", "mlflow", "/bin/bash", "-c"]
 
 ARG TRITON_DIR=/mlflow/triton-inference-server
-ARG TRITON_VER=r22.10
+ARG TRITON_VER=r23.01
```
Comment (mdemoret-nv) [inline models/mlflow/docker/Dockerfile:36]: Does this require an update to all locations where the triton version is specified?

**Review by mdemoret-nv — APPROVED** [review]
Just one comment and then we can merge.

**Review by dagardner-nv — APPROVED** [review]

File: models/mlflow/docker/Dockerfile (hunk 1 above)
Comment (mdemoret-nv) [reply models/mlflow/docker/Dockerfile:36]: Resolving since this is only to the mlflow dockerfile.

**Pull request merged**